modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -945,6 +946,121 @@ public String getTypeDeclaration(Schema p) {
         return super.getTypeDeclaration(target);
     }
 
+    private String getBeanValidation(Schema<?> items) {
```

Comment:
minor suggestion: add a docstring for this method

Reply:
done